### PR TITLE
add set_random_seed to the python interface

### PR DESCRIPTION
**Pull request by nitnelave**
There is currently (AFAIK) no way to set the random seed, making it impossible to have reproducible results.

### CODE REVIEW

File: python/caffe/_caffe.cpp [hunk 1]
```diff
@@ -51,6 +51,9 @@ const int NPY_DTYPE = NPY_FLOAT32;
 void set_mode_cpu() { Caffe::set_mode(Caffe::CPU); }
 void set_mode_gpu() { Caffe::set_mode(Caffe::GPU); }
 
+// Set the random seed
```
Comment (longjon) [inline python/caffe/_caffe.cpp]: Could you remove this redundant comment?

**Comment by longjon** [comment]
Looks good except as noted, thanks.

**Comment by nitnelave** [comment]
Comment removed as noted.

**Comment by shelhamer** [comment]
Thanks for extending the Python interface @nitnelave.

**Pull request merged**